cmd/destroy/cmddestroy.go

```diff
@@ -64,9 +66,20 @@ func (r *DestroyRunner) RunE(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	// Retrieve the inventory object.
+	reader := r.provider.ManifestReader(cmd.InOrStdin(), args)
+	infos, err := reader.Read()
+	if err != nil {
+		return err
+	}
+	inv, _, err := inventory.SplitInfos(infos)
+	if err != nil {
+		return err
+	}
+
 	// Run the destroyer. It will return a channel where we can receive updates
 	// to keep track of progress and any issues.
-	ch := r.Destroyer.Run()
+	ch := r.Destroyer.Run(inv)
```

Comment:
I think this is an example where if we could just let the command accept the information needed to look up the inventory object, that would be a better api.